.travis.yml

```diff
@@ -2,23 +2,49 @@ language: php
 
 matrix:
   include:
-    - php: 5.3
     - php: 5.4
     - php: 5.5
     - php: 5.6
+    - php: 7
+    - php: nightly
```

Comment:
php 7 en nightly zijn eigenlijk exact hetzelfde in travis. Die zullen momenteel wss allebei de alpha inladen (als travis al upgedate heeft naar de laatste versie natuurlijk)